CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java

```diff
@@ -171,6 +179,7 @@ public static synchronized int start() {
                 final boolean pythonRestClientDownload = prefs.getBoolean(ApplicationPreferenceKeys.PYTHON_REST_CLIENT_DOWNLOAD, ApplicationPreferenceKeys.PYTHON_REST_CLIENT_DOWNLOAD_DEFAULT);
                 if (pythonRestClientDownload) {
                     downloadPythonClient();
+                    installPythonPackage();
```

Comment:
is `downloadPythonClient()` still needed with this new approach? I think it might be able to be removed (since that was the old way of adding the client)